data/json/recipes/recipe_weapon.json

```diff
@@ -195,6 +195,32 @@
       [ [ "duct_tape", 20 ], [ "filament", 20, "LIST" ] ]
     ]
   },
+  {
+    "type": "recipe",
+    "result": "spear_stone",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_PIERCING",
+    "skill_used": "fabrication",
+    "skills_required": [ "survival", 1 ],
+    "difficulty": 2,
+    "time": 2400,
+    "autolearn": true,
+    "qualities": [ { "id": "CUT", "level": 1 }, { "id": "HAMMER", "level": 1 } ],
+    "components": [
+      [ [ "stick_long", 1 ], [ "pool_cue", 1 ] ],
+      [ [ "rag", 1 ], [ "felt_patch", 1 ], [ "leather", 1 ], [ "fur", 1 ] ],
+      [
+        [ "plant_fibre", 20 ],
+        [ "sinew", 20 ],
+        [ "thread", 20 ],
+        [ "yarn", 20 ],
+        [ "duct_tape", 20 ],
+        [ "resin_cord", 1 ],
```

Comment:
`resin_cord` is also a MST mod item. It needs to be removed from the stone spear recipe in order for the base game not to throw errors.

Reply:
Thanks